### ISSUE TITLE
Implement showing stats in uploads page?

### ISSUE THREAD

**Issue by mlnl221** [slskd/slskd#1149]
I would like to be able to see my stats displayed on the upload side.

I see there is code to retrieve stats, but nothing has been implemented in react to display it. Could it be possible to build a small component that rends out the stats when clicked? It would be keep track of my user stats on the soulseek server.

This is something that I use quite frequently on nicotine+

**Comment by molnarti** [comment]
seconded. i find the lack of "reward" in form of some simple statistics for making all my hoarded stuff accessible for everyone a bit frustrating...

**Comment by GabrielPearce** [comment]
Yes I would Love a feature like this - just good to know that your shares are working, and satisfaction of course

**Comment by Byrnesdigital** [comment]
I just want timestamps but I would take more detailed stats as well
